kotlin/edit-features-with-feature-linked-annotation/src/main/java/com/esri/arcgisruntime/sample/editfeatureswithfeaturelinkedannotation/MainActivity.kt

```diff
@@ -136,7 +138,19 @@ class MainActivity : AppCompatActivity() {
         (result.layerContent as? FeatureLayer)?.let { featureLayer ->
           // get a reference to the identified feature
           selectedFeature = result.elements[0] as? Feature
-          selectedFeature?.also {
+          // if the selected feature is a polyline with more than one segment
+          if ((selectedFeature?.geometry is Polyline)
+            && (selectedFeature?.geometry as Polyline).parts[0].pointCount > 2
```

Comment:
This seems like a fine solution and I think I checked it with every curved polyline in the sample, but is it technically possible for a curved polyline to have exactly two points in its first part but more points in other parts?

Reply:
Hmm...that's a very good point...